### DISCUSSION TITLE
Getting error at the time of displaying calculated value in grid

### DISCUSSION THREAD

**Discussion by nehagaur89** [pimcore/pimcore#17880] [Admin-Ui-Classic-Bundle]
I am getting an error when I am trying to display number in calculated value in grid options.  We need to convert the value to String to fix this error. Can somebody push it. thanks 
![Screenshot from 2024-12-02 12-45-59](https://github.com/user-attachments/assets/1a8ebdaa-2631-4de9-a848-15bb7bdf5de4)

 TypeError: value.replace is not a function
    at klass.<anonymous> (script-proxy?storageFile=minified_javascript_core_8c13b00b48771c84bf4d4dde18f7774b.js&_dc=b077b49d3572660d29109bf445e3aed8e0325e21:94953:33)

**Comment by kingjia90** [comment]
Thank you for reporting!
You are likely using an older pimcore version (< 11), because it was fixed [here](https://github.com/pimcore/admin-ui-classic-bundle/commit/cd696fb58464e3e88f8727d0f10a1e4f3fc189c4#diff-bd800665eab789c6227dd9e92e7f85c39cf07bd5b33a40c8ad3ad039af0170fdL105). 

Let me know if the problem still occurs in the latest
